src/llmcompressor/modifiers/transform/spinquant/base.py

```diff
@@ -219,6 +227,13 @@ def _create_r2_scheme(self, model: PreTrainedModel) -> TransformScheme:
         else:
             raise NotImplementedError()
 
+        if self.hadamard_size:
+            assert (
```

Comment:
Can you change this to a ValueError? Imho asserts should never trigger in practice, while ValueErrors are used to signal something wrong about the user's configuration.

Reply:
updated